Dockerfile

```diff
@@ -1,13 +1,45 @@
-FROM golang:1.13
-MAINTAINER Conjur Inc
+FROM golang:1.13 as secrets-provider-builder
+MAINTAINER CyberArk Software, Ltd.
 
-RUN apt-get update && apt-get install -y jq
+ENV GOOS=linux \
+    GOARCH=amd64 \
+    CGO_ENABLED=0
+
+COPY . /opt/secrets-provider-for-k8s
+
+WORKDIR /opt/secrets-provider-for-k8s
+
+EXPOSE 8080
+
+#RUN apt-get update && apt-get install -y jq
```

Comment:
what's this commented-out line?

Reply:
missed that in between commits. Should be updated now